### PR TITLE
Enable Github Enterprise

### PR DESCRIPTION
**Pull request by elsayed-atef**
this PR is to enable the library to work on GHE

### CODE REVIEW

**Comment by sotayamashita** [comment]
@elsayed-atef Awesome :smile: I would like to confirm with GHE, is there any way to do for free?

File: index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const debug = require("debug")("probot:pr-triage");
 const Raven = require("raven");
 const PRTriage = require("./lib/pr-triage");
+const getConfig = require('probot-config');
```
Comment (sotayamashita) [inline index.js]: Do you need to add `probot-config`? It seems that it does not use it.

Comment (sotayamashita) [inline index.js]: Do you need to add `probot-config`? It seems that you do not use it.

**Comment by elsayed-atef** [comment]
@sotayamashita There's no fees from GHE side , once you create a github App and make your standalone deployment , you can use it

File: package.json [hunk 2]
```diff
@@ -54,7 +54,8 @@
     "smee-client": "^1.0.1",
     "sync-glitch-cli": "^2.0.0",
     "travis-deploy-once": "^5.0.0",
-    "validate-commit-msg": "^2.14.0"
+    "validate-commit-msg": "^2.14.0",
+    "probot-config": "^0.1.0"
```
Comment (sotayamashita) [inline package.json]: Please remove `probot-config` too.

File: .env.example [hunk 3]
```diff
@@ -7,3 +7,7 @@ LOG_LEVEL=debug
 
 # Go to https://smee.io/new set this to the URL that you are redirected to.
 WEBHOOK_PROXY_URL=
+
+#NODE_ENV=production
+#GHE_HOST=Github-Enterprise-Domain
+#PRIVATE_KEY_BASE64="$(cat ./*.private-key.pem | base64)"
```
Comment (sotayamashita) [inline .env.example]: [Question] Is it needed?

File: .env.example [hunk 4]
```diff
@@ -7,3 +7,7 @@ LOG_LEVEL=debug
 
 # Go to https://smee.io/new set this to the URL that you are redirected to.
 WEBHOOK_PROXY_URL=
+
+#NODE_ENV=production
```
Comment (sotayamashita) [inline .env.example]: [Question] Is it needed?

File: package.json (hunk 2 above)
Comment (elsayed-atef) [reply package.json]: @sotayamashita if removed it won't support GHE

File: .env.example (hunk 3 above)
Comment (elsayed-atef) [reply .env.example]: yes , you should provide your GHE name and also the private generated from the github app for authentication

Comment (elsayed-atef) [reply .env.example]: https://github.com/probot/probot/blob/master/docs/deployment.md

File: .env.example (hunk 4 above)
Comment (elsayed-atef) [reply .env.example]: https://github.com/probot/probot/blob/master/docs/deployment.md

File: .env.example (hunk 3 above)
Comment (sotayamashita) [reply .env.example]: @elsayed-atef I got it. Thanks

**Comment by sotayamashita** [comment]
@elsayed-atef Sorry to be late. I will confirm how it works on GHE on this weekend.

**Pull request merged**